Plan/src/main/java/com/djrapitops/plan/command/commands/manage/ManageBackupCommand.java

```diff
@@ -93,7 +93,7 @@ public void onCommand(Sender sender, String commandLabel, String[] args) {
 
             String dbName = args[0].toLowerCase();
 
-            boolean isCorrectDB = Verify.equalsOne(dbName, "sqlite", "mysql");
+            boolean isCorrectDB = Verify.equalsOne(dbName, "sqlite", "mysql", "h2");
             Verify.isTrue(isCorrectDB,
                     () -> new IllegalArgumentException(locale.getString(ManageLang.FAIL_INCORRECT_DB, dbName)));
```

Comment:
`ManageLang.FAIL_INCORRECT_DB` may require changes to include h2 in the list of accepted arguments

Reply:
Not required, the message is "> §c'${0}' is not a supported database."